src/main/java/org/broadinstitute/hellbender/tools/spark/sv/SVKmer.java

```diff
@@ -43,7 +44,7 @@
      *  Makes an empty SVKmer.  If you call toString on it, it'll look like poly-A.
      */
     public SVKmer( final int kSize ) {
-        if ( kSize < 1 || kSize > 63 ) throw new IllegalArgumentException("K must be between 1 and 63.");
+        Utils.validateArg(kSize >= 1 && kSize <= 64, "K must be between 1 and 63.");
```

Comment:
that suggests that there's a missing test for value 64. Can you add such a test to confirm that this check blows up on it?